charts/authz/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```

Comment:
I think you also need to add value folder for env value files. So to make it more clear if we compare helm to python helm chart as a Class and values for different envs are used to initialize a specific object of that class. Objects from one environment to another (for example prod/dev) will differ because they will most likely have different configurations. Because of this we need different value files

Reply:
Worked on all the comments. I hope...